### DISCUSSION TITLE
02230307

### DISCUSSION THREAD

**Discussion by Eyemusican** [kevinlin1/just-the-class#43] [Q&A]
https://www.figma.com/file/hiDway6P8SeZ0V2fwfUHDj/Untitled?type=design&node-id=0-1&mode=design&t=XkerAGyprmX8fS2a-0

**Comment by kevinlin1** [comment]
Hi, I think you meant to post this in the [discussion for your course](https://github.com/rubcstswe/sda101/discussions). This discussion is for the website template, not any particular course.
